public/assets/css/general.css

```diff
@@ -516,18 +516,23 @@ table {
 	display: block;
 	width: 100%;
 }
-
-#video-player {
+#video-player, .showlite-asset {
+    padding: 6px;
+    border: 1px solid #555;
     background-color: #fff;
+}
+#video-player {
     cursor: pointer;
-    margin: 0 auto;
-    margin-top: 2%;
-    border: 1px solid #d0d0d0;
-    padding:6px;
 }
 
 .showlite-asset {
     max-width: 50%;
+    margin: 0 auto;
+    margin-top: 2%;
+    padding: 6px;
+    border: 1px solid #a2a2a2;
```

Comment:
I think I prefer the images without a border., but also the `margin: 0 auto;` doesn't seem to be taking affect.  I'd like to see what it looks like if the images are centered like the video.